frontends/web/src/components/terms/btcdirect-otc-terms.tsx

```diff
@@ -26,26 +26,27 @@ type TProps = {
   onContinue: () => void;
 }
 
+export const getBTCDirectPrivacyLink = () => {
+  switch (i18n.resolvedLanguage) {
+  case 'de':
+    return 'https://btcdirect.eu/de-at/datenschutzenklaerung?BitBox';
+  case 'nl':
+    return 'https://btcdirect.eu/nl-nl/privacy-policy?BitBox';
+  case 'es':
+    return 'https://btcdirect.eu/es-es/privacy-policy?BitBox';
+  case 'fr':
+    return 'https://btcdirect.eu/fr-fr/privacy-policy?BitBox';
+  default:
+    return 'https://btcdirect.eu/en-eu/privacy-policy?BitBox';
+  }
+};
```

Comment:
The other links come from routes/exchange/components/infocontent.tsx

but it seems completely wrong to import from routes here.

I considered moving terms into routes/exchanges, but one of the terms is also used for routes/bitsurance, so I just left it as is now.